### PR TITLE
Add relative to target orbital information

### PR DESCRIPTION
**Pull request by pacbard**
Variables are visible only target is selected. The same information is available visually in the map view. All calculations are inspired by Kerbal Engineer's rendezvous calculator.

Documentation is also updated to reflect the new changes.

### CODE REVIEW

File: doc/globals.mkd [hunk 1]
```diff
@@ -133,4 +133,7 @@ Craft.Inter2.Δt  | Same as Craft.Inter1.dt
 Craft.Inter2.Sep | Distance (in meters) between our craft and the target at Craft.Inter2.dt
 Craft.Inter2.TrueAnomaly | True anomaly of the second time we cross the target orbit
 Craft.Inter2.θ   | Same as Craft.Inter2.TrueAnomaly
+Craft.Rel.AN     | Argument of Ascending node relative to target
+Craft.Rel.DN     | Argument of Descenging node relative to target
+Craft.Rel.Inc    | Incliation relative to target
```
Comment (wmvanvliet) [inline doc/globals.mkd]: typo: Inclination

**Comment by wmvanvliet** [comment]
could you update `CHANGES.md` too?

**Comment by wmvanvliet** [comment]
the globals work fine by the way, thanks!

**Comment by pacbard** [comment]
Done!

**Comment by wmvanvliet** [comment]
Thanks!

**Pull request merged**